From af5ec42a8a380be17ea46f2270b119e7e6c5792f Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 24 May 2022 13:26:12 +0300
Subject: [PATCH] [SW-71880] gaudi2: remove u-boot version reg usage

Starting from Gaudi2, the mmUBOOT_VER_OFFSET register is not used
anymore. The version passing is done through the COMMs protocol, and/or
the FW status packet of the FW_UPD protocol.
Remove the usage of the register.

Change-Id: I024cc5b457c213eebf5ac694d6a07384f46e4e63
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi2/common/common.c b/arch/gaudi2/common/common.c
index a5ece51..9885e08 100644
--- a/arch/gaudi2/common/common.c
+++ b/arch/gaudi2/common/common.c
@@ -183,7 +183,7 @@
 	regs->fw_upd_pending_sts = mmUPD_PENDING_STS;
 	regs->fuse_ver_offset = 0;	/* unused */
 	regs->preboot_ver_offset = 0;	/* unused */
-	regs->uboot_ver_offset = mmUBOOT_VER_OFFSET;
+	regs->uboot_ver_offset = 0;	/* unused */
 	regs->hw_state = mmHW_STATE;
 	regs->kmd_msg_to_cpu = mmPSOC_GLOBAL_CONF_KMD_MSG_TO_CPU;
 	regs->cpu_cmd_status_to_host = mmCPU_CMD_STATUS_TO_HOST;
